_posts/2018-07-25-strimzi-0-5-0-release.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Strimzi 0.5.0 released!"
+date: 2018-07-25
+author: paolo_patierno
+---
+
+We are really happy to announce the new Strimzi 0.5.0 release with a lot of new awesome features!
+
+<!--more-->
+
+# Kafka as a native Kubernetes/OpenShift resource
+
+The first really cool feature is about the Cluster Operator moving from ConfigMaps to Custom Resources.
+It means that the Kafka and the Kafka Connect clusters aren't described by ConfigMaps anymore but through Custom Resources.
+Here's an example of the `Kafka` resource for deploying a Kafka cluster:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      type: ephemeral
+...
+...
+zookeeper:
+    replicas: 3
+...
+```
+
+Kafka is now a first citizen in the Kubernetes/OpenShift world and the cluster deployment can be managed through the `kubectl` and `oc` tools natively.
+For example, you can get Kafka cluster information through the following command:
+
+```
+oc describe kafka my-cluster
+```
+
+So `kafka` is now at same level as any other Kubernetes/OpenShift native resourse, for example `configmap`, `deployment`, `secret` and so on.
+The same is true for Kafka Connect cluster which is defined by a `KafkaConnect` Custom Resource.
```

Comment:
When you talk abotu specific resources, you should mention also `KafkaConnectS2I` resource.